### PR TITLE
Added button to exit customization page

### LINKED ISSUE #69: 🚀 Feature Request: Create a button that exits the customization page
### Problem Overview 👁️‍🗨️ 

Users cannot exit the customization screen before completing all the options
### What would you like? 🧰

An exit button to the left of the "Next" button on the customized page
### What alternatives have you considered? 🔍

I thought of adding an exit button to the top left, but there is already a customization progress indicator on the top of the screen
### Additional details ℹ️

NA

### PR DESCRIPTION
**Pull request by soumyankar**
Fixes #69 
I hope the design of the button is satisfactory.

### CODE REVIEW

File: packages/app/src/pages/CustomizePracticePage.vue [hunk 1]
```diff
@@ -35,10 +35,20 @@
...
-          <q-stepper-navigation class="justify-center">
+          <q-stepper-navigation class="flex justify-center q-gutter-sm">            
+            <router-link
+              to="/"
+            >
+            <q-btn
+                class=""
+                rounded
+                color="red"
+                label="Stop"
```
Comment (milindvishnoi) [inline packages/app/src/pages/CustomizePracticePage.vue:46]: Please change the label to "close".

**Review by milindvishnoi — CHANGES_REQUESTED** [review]
Great job on the PR!

Please also increase the width of both the buttons, they look a bit too small.

**Comment by soumyankar** [comment]
I've changed the button sizes and changed the label as well! 😄

**Comment by ArsalaBangash** [comment]
@soumyankar I think 'exit' would be a better label than close :)

**Comment by soumyankar** [comment]
> @soumyankar I think 'exit' would be a better label than close :)

Done!

**Review by ArsalaBangash — APPROVED** [review]
Excellent!

**Pull request merged**